pyproject.toml

```diff
@@ -1,18 +1,20 @@
 [tool.poetry]
 name = "flake8_per_file"
-version = "0.5.0"
+version = "0.5.1"
 description = "Per file flake8"
 authors = ["Gavin Carothers <gavin@carothers.name>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.1"
```

Comment:
What's the bump here for?

Reply:
`isort` advertises itself as requiring python 3.6.1 and newer for some reason.

Reply:
Weird. No comments otherwise.